src/tabs/Pixnflix/index.tsx

```diff
@@ -84,34 +90,34 @@ class PixNFlix extends React.Component<Props, State> {
     }
   }
 
-  private onClickStill = () => {
-    const { mode } = this.state;
-    if (mode === ('still' as SideContentVideoDisplayMode)) {
-      this.handleSnapPicture();
-    } else {
-      this.swapModes(mode)();
-    }
-  };
-
   public setupVideoService = () => {
     if (this.$video && this.$canvas && this.isPixNFlix()) {
       const { debuggerContext } = this.props;
       this.pixNFlix = debuggerContext.result.value;
-      // get the properties of the video in an array
-      // [height, width, FPS]
-      const videoProperties: number[] = this.pixNFlix.init(
+      // get the properties of the video in an object
+      // { Height, Width, FPS, VOLUME, inputFeed }

```

Comment:
Comment seems a bit outdated wrt to the names, though it could potentially be removed, and the constants deduced from the return value of `init()` (`return { HEIGHT, WIDTH, FPS, VOLUME, inputFeed };`)